recipes/natsort/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "natsort" %}
+{% set version = "4.0.4" %}
```

Comment:
The current version of `natsort` is `5.0.2` released January 27, 2017. Please update the recipe to reflect the most recent version:

https://pypi.python.org/pypi/natsort